src/main/java/yesman/epicfight/world/capabilities/item/CapabilityItem.java

```diff
@@ -92,6 +95,12 @@ public static List<AttributeModifier> getAttributeModifiers(Attribute attribute,
 		
 		return attributeModifiers;
 	}
+
+	private static boolean validateAttribute(LivingEntityPatch<?> patch, Attribute instance)
+	{
+		AttributeInstance attributeInstance = patch.getOriginal().getAttribute(instance);
+		return attributeInstance != null;
```

Comment:
Suggested by @Yesssssman 

```suggestion
		return patch.getOriginal().hasAttribute(instance);
```